feat: Add the Linked Wearables V2 documentation (#449)
decentraland/documentation: content/creator/wearables-and-emotes/wearables/linked-wearables.md

All Linked Wearables are defined inside of a **Linked Wearable Collection**. We'll see how to create one further down into the article.

## How do Linked Wearables represent NFTs?

Wearables are linked to your NFTs by creating a Linked Wearable Collection in the [Builder site](https://decentraland.org/builder) and setting how your NFTs will be represented at the time of creating the wearables.

We support 4 mechanisms to link your wearables to NFTs. All of these mechanisms use the token id of the NFTs to match them.

The following table shows the mentioned mechanisms:

| Match         |                                                                                                                                     |
| ------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
| All NFTs      | A user owning any of the NFTs of the collection will own the wearable                                                               |

> | Single NFT    | A user owning the specified TOKEN ID will own the wearable. _e.g. 123456_                                                           |

Reviewer: ```suggestion
| Single NFT    | A user owning an NFT specified by TOKEN ID will own the wearable. _e.g. 123456. The user will own the wearable if they own the NFT with TOKEN ID: 123456_                                                           |
```

> | Multiple NFTs | A user owning one of the multiple TOKEN IDs, described as separated by a comma will own the wearable. _e.g. 123456, 123457, 123458_ |

Reviewer: ```suggestion
| Multiple NFTs | A user owning one of many NFTs specified by TOKEN IDs, described as separated by a comma will own the wearable. _e.g. 123456, 123457, 123458. The user will own the wearable if they own the NFT with TOKEN ID: 123456 or 123457 or  123458_ |
```

> | Range of NFTs | A user owning an NFT with a token id in the range of TOKEN IDs described, will own the wearable. _e.g. [1, 1000]_                   |

Reviewer: ```suggestion
| Range of NFTs | A user owning one of many NFTs specified in the range of TOKEN IDs will own the wearable. _e.g. [1, 1000]. The user will own the wearable if they own the NFT with TOKEN ID 1 or 2 or 3 or 4 or 5 ... or 100_                   |
```

   ![](/images/media/linked-wearables/multiple-files-upload-correct-file.png)

6. Upload all wearables by clicking **Upload items**.

   ![](/images/media/linked-wearables/edit-multiple-files-without-errors.png)

7. Be patient, this might take a while!

   ![](/images/media/linked-wearables/edit-multiple-files-uploading.png)

8. Success! Your items are now available in your collection. Check the forum post for any updates from the curator.

   ![](/images/media/linked-wearables/edit-multiple-files-success.png)

# Publishing your Linked Wearables

Your Linked Wearables need to go through a publishing and curation process as the regular wearables do. Although the publication and curation process is not the same as the one for the regular wearables, it keeps the same steps, all items must be first be published to later be curated by an assigned curator.

The following sections will show you how to publish your Linked Wearables to be curated.

## Costs

Creating Linked Wearables has a cost depending on the type of Linked Wearable Collection you chose to build:

> 1. **Standard**: each published wearable costs the same as a regular wearable.

Reviewer: ```suggestion
1. **Standard**: each published wearable costs the same as a regular wearable: USD 100 payable in MANA.
```

> 2. **Programmatic**: a fixed fee, payed once for all the wearables you'll be publishing.

Reviewer: ```suggestion
2. **Programmatic**: a fixed fee, payed once for all the wearables you'll be publishing. This fee equals publishing 20 regular wearables: 2000 USD payable in MANA.
```

   ![](/images/media/linked-wearables/sign-items-confirmation.png)

5. Read and check the Terms and Conditions.

   ![](/images/media/linked-wearables/terms-and-conditions.png)

6. Check your publishing fees. The fees required for published the wearables are described in the [Costs section](#costs).

   ![](/images/media/linked-wearables/publishing-payment.png)

7. If it's your first time publishing Linked Wearables, you'll need to authorize the Linked Wearables smart contract to operate MANA on your behalf. This step is needed to deduct the MANA used to pay the publication fees from your wallet.

   ![](/images/media/linked-wearables/first-time-publishing.png)

8. Pay the publication fee and complete the publishing by performing the transaction. Depending on the congestion of the network, this might take a while.

   ![](/images/media/linked-wearables/complete-publishing.png)

9. Success! You have published your items. Your items will go through the curation process a regular collections do. You can communicate with the curator via the forum post.

   ![](/images/media/linked-wearables/publication-success.png)

## Pushing changes for review

> Previously published and approved wearables that are edited need to go through the curation process again. Don't worry, there won't be any fees applied to already published wearables.

Reviewer: ```suggestion
Published and approved wearables that are edited need to go through the curation process again. Don't worry, there won't be any fees applied to already published wearables.
```
